### DISCUSSION TITLE
黄粱五载梦 | /cn/2024-01-18-/

### DISCUSSION THREAD

**Discussion by giscus** [JinjiPang/jinji#54] [Comments]
# cn/2024-01-18-/

人生中的又一个五年结束了，2018-2023。于是乎，当年那个高考结束后才匆匆填报了一所以前从未听过的学校和专业的我，从动物医学这个专业一路读到了博士。
不夸张的说，我在Ames度过了美好的五年，有时日子过得比本科时的自己还要清闲自在。博一、博二多数时间是在上专业课，少数时间在实验室做实验，博三开始了统计学硕士学位的学习，于是乎就又上了两年课，直到博四结束才上完所有的课。上课、写作业、考试经历的太 &hellip;

https://jinjipang.com/cn/2024-01-18-/

**Comment by yihui** [comment]
我居然出现在博士论文致谢里了，不胜荣幸。

只可惜你上次来的时候我还无暇研究 K 歌设备，过了几个星期之后才捣鼓；下次要是再来可以欢唱一下。《加州招待所》也是我比较喜欢听和唱的曲目。其实我离职的公告里是很想写“we are prisoners of our own device”的，也就是人为自己创造的工具所累，但这话尺度太大，不能乱讲。

我突然意识到你这羽毛球进步的神速很可能跟攀岩也有关系，因为初学者的第一个进步障碍就是小臂肌肉无力，这会导致无法打出短促鞭打的感觉，必须要特别加强握力和小臂力量的训练。对厨子来说，通过手腕和小臂急速把肉或鱼剁成酱就是很好的训练；如果不剁鱼肉，也可以用快速捏喷水壶训练。我平时候有时候会剁鱼，而春天来了在草坪上打药时会捏药壶，这两件事都是高强度的锻炼。有时候我跟球友们说剁刀有助于提高羽毛球技术，他们以为我在开玩笑。

最后祝贺顺利毕业！启功老爷子的话又可以搬出来用了：入学初识门庭，毕业非同学成。涉世或始今日，立身却在生平。

**Reply by hongtaoh** [reply]
是想有机会开车过去拜访益辉的，但看到你在一个帖子留言中说每天工作 16 个小时，想想还是算了吧

**Reply by yihui** [reply]
仅仅是最近几周，忙到冒烟，这肯定不会成为常态。欢迎你来，可以考虑暑假里来。

**Reply by hongtaoh** [reply]
好嘞！

**Comment by hongtaoh** [comment]
真的是一条道最到黑。我们都在羡慕别人，我还羡慕你终于博士毕业、能去工作、终于去了大城市。

**Reply by JinjiPang** [reply]
我理解你，每一个没有毕业的博士最羡慕的都是已经毕业了的人。作为一个过来人，可能我要说的就是活在当下吧。

**Reply by hongtaoh** [reply]
还有坚持。
